source/examples/rapids-sagemaker-higgs/notebook.ipynb

```diff
@@ -353,7 +353,7 @@
     }
    ],
    "source": [
-    "!docker build -t {estimator_info['ecr_image']} --build-arg RAPIDS_IMAGE={{ rapids_container }} ."
+    "!docker build -t {estimator_info['ecr_image']} --build-arg RAPIDS_IMAGE={estimator_info['rapids_container']} ."
```

Comment:
`{{ rapids_container }}` is already templated into this `estiamtor_info` dictioanry a few lines up.

Just using a dictionary reference here instead of repeating the templating reduces the risk of inconsistencies, and makes interactive testing (where I found it convenient to `git clone` the repo and manually replace these with different image tags) a bit easier.